src/pages/Transaction/TransactionCell/index.tsx

```diff
@@ -140,6 +143,84 @@ const TransactionCellIndexAddress = ({
   )
 }
 
+const DOBInfo: FC<{ cell: Cell$Spore }> = ({ cell }) => {
+  const [imageUrl, setImageUrl] = useState('')
+  const { collection: { typeHash } = {}, tokenId = '' } = cell.extraInfo
+  const { data } = useQuery(
+    ['nft-item-info', typeHash, tokenId],
+    () => explorerService.api.fetchNFTCollectionItem(typeHash!, tokenId),
+    {
+      enabled: !!typeHash && !!tokenId,
+    },
+  )
+  const tokenIdStr = `${data?.standard === 'spore' ? '' : '#'}${formatNftDisplayId(
+    data?.token_id ?? '',
+    data?.standard ?? null,
+  )}`
+
+  useEffect(() => {
+    getSporeImg({ data: cell.extraInfo.data, id: cell.extraInfo.tokenId }).then(res => {
+      setImageUrl(res)
+    })
```

Comment:
Fallback of the dob image can be added, e.g. https://github.com/nervosnetwork/ckb-explorer-frontend/blob/develop/src/utils/util.ts#L190-L198